content/W2016/W16-event-teaparty.md

```diff
@@ -0,0 +1,24 @@
+title: Tea Party and Volunteer Meeting
+Date: 2015-12-18
+Category: Events
+Tags: social, volunteer, big csters
+Slug: tea-party-w16
+Author: Fatema Boxwala
+Summary: Join us for tea and for information on volunteering with us!
+
+Welcome back! You're cordially invited to our Tea Party and Volunteer meeting!
+Come out to socialize with some WiCS members, meet the new committee, and learn
+more about our volunteer opportunities! The volunteer meeting will run from
+5:00 - 5:30, and the Tea Party will go from 5:30 to 7:00. If you just want to
+come for the tea party, feel free to come by at 5:30! Please try to bring your
```

Comment:
Hating to nitpick further, but there are like 4 sentences ending in '!' in a row. Seems to be a bit overly-enthused :P